protocol/0013-ACCT-accounts.md

```diff
@@ -17,14 +17,16 @@ Note that a party can also associate the governance / staking asset via the [Veg
 1. Mark-to-market settlement account per market: this is used for collecting and distributing mark-to-market settlement cashflows and is *zero* at the end of each mark-to-market settlement run.
 1. Margin accounts for each party with open orders or positions on any [market](./0043-MKTL-market_lifecycle.md).
 1. Bond account for any party that's an [LP on any market](0044-LIME-lp_mechanics_type.md).
-1. [Insurance pool account](0015-INSR-market_insurance_pool_collateral.md) for any market.
+1. [Global insurance pool](0015-INSR-market_insurance_pool_collateral.md#global-insurance-pool) (1 per asset)
+1. [Insurance pool account](0015-INSR-market_insurance_pool_collateral.md#market-insurance-pool) for any market.
 1. [Liquidity fee pool](0042-LIQF-setting_fees_and_rewarding_lps.md) for any market.
 1. [Infrastructure fee pool](0029-FEES-fees.md) for any asset.
-1. [Reward accounts](0056-REWA-rewards_overview.md) which exist for *each* reward account per every Vega asset (settlement asset) and per every reward metric per every Vega asset (reward asset). There is an additional [staking rewards](0061-REWP-pos_rewards.md) account.
-1. [Vega trasury accounts](0055-TREA-on_chain_treasury.md) per Vega asset.
+1. [Reward accounts](0056-REWA-rewards_overview.md) which exist for *each* reward account per every Vega asset (settlement asset) and per every reward metric per every Vega asset (reward asset). There is an additional [global rewards account](0056-REWA-rewards_overview.md#validator-ranking-metric) used for supplementary (on top of infrastructure fee split) validator rewards.
 
 One key difference with staking accounts is that the collateral is not held in an asset bridge, but in the [staking bridge](./0071-STAK-erc20_governance_token_staking.md). The balance is changed by events on Ethereum, rather than actions taken on the Vega chain.
 
+Note that both the global insurance pool and rewards account use the same `0` address. Account type is used to differentiate where the funds should go into when making a transfer or deposit.
```

Comment:
it's how it's implemented I don't know if it's what's required.

Reply:
Thanks, I wanted to capture what's implemented for now and get a review from research so they can flag up anything that they'd like to change.

Reply:
As long as they both work as expected this seems fine to me.

Reply:
Ok, added ACs to assure both deposits and transfers work.